bundle/LegacyMapper/SiteAccess.php

```diff
@@ -47,6 +47,10 @@ public function onBuildKernelWebHandler(PreBuildKernelWebHandlerEvent $event)
         $request = $event->getRequest();
         $uriPart = array();
 
+        // @todo Remove when, if ever, legacy does not use deprecated constructor names
+        // triggers initial autoload of eZSiteAccess and sileces deprecation warning
+        @eZSiteAccess::TYPE_DEFAULT;
```

Comment:
I don't get this...
As for the comment, you just need to silent `E_DEPRECATED` errors in php.ini